python/az/aro/azext_aro/_validators.py

```diff
@@ -70,6 +70,15 @@ def validate_domain(namespace):
                            namespace.domain)
 
 
+def _validate_int(key, i):
```

Comment:
this is re-adding something that should be being removed.

Reply:
I can remove this.  This was added to the azure/azure-cli repository.

Reply:
I think it was removed here with the intention of updating azure/azure-cli and removing it there too.